processing/data_collection/gazette/spiders/rj_campos_goytacazes.py

```diff
@@ -0,0 +1,67 @@
+import re
+import dateparser
+
+from datetime import datetime
+from scrapy import Request
+from scrapy.exceptions import CloseSpider
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjCampoGoytacazesSpider(BaseGazetteSpider):
+    GAZETTE_ELEMENT_CSS = "ul.ul-licitacoes li"
+    TERRITORY_ID = "3301009"
+
+    allowed_domains = ["www.campos.rj.gov.br"]
+    name = "rj_campos_goytacazes"
+    start_urls = [
+        "https://www.campos.rj.gov.br/diario-oficial.php?PGpagina=1&PGporPagina=15"
+    ]
+    LIMIT_DATE = dateparser.parse("2015-01-01").date()
```

Comment:
**suggestion**: If you are creating a `datetime.date` object, there is no need to use an intermediary like `dateparser`.

[Built-in date](https://docs.python.org/3/library/datetime.html#datetime.date) from Python is enough:

`LIMIT_DATE = datetime.date(2015, 1, 1)`

Reply:
I'll address this once we settle on https://github.com/okfn-brasil/querido-diario/pull/235#discussion_r491440601.

Reply:
I think you forget to update this line. Because I cannot see the change in the highlighted comment. Am I wrong?

Reply:
I did not update this line because I was waiting on a reply on my question in https://github.com/okfn-brasil/querido-diario/pull/235#discussion_r491440601. I'll update it now!